React/tic-tac-toe/src/redux/Game/reducer.js

```diff
@@ -0,0 +1,20 @@
+import * as types from './actions';
+
+const initialState = {
+  stepNumber: 0,
+  xIsNext: true
+};
+
+const reducerGame = (state = initialState, action) => {
+  switch (action.type) {
+    case types.CHANGE_STEP:
+      return Object.assign({}, state, {
+        stepNumber: action.number,
+        xIsNext: !state.xIsNext
```

Comment:
I think you should use action.bool here. This may break when going to a previous step.